src/core/Slider/Slider.test.tsx

```diff
@@ -714,3 +714,18 @@ it('should activate thumb on pointerDown and move to closest step on move/ no up
   expect(thumbs[0].getAttribute('aria-valuenow')).toEqual('40');
   expect(thumbs[1].getAttribute('aria-valuenow')).toEqual('80');
 });
+
+it('should not break pointer events when thumb is released', () => {
+  const { container } = render(
+    <Slider min={0} max={100} values={[20, 80]} step={1} />,
+  );
+
+  assertBaseElement(container);
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).not.toHaveClass('iui-active');
+
+  act(() => {
+    // userEvent.pointer is not released yet 😕
```

Comment:
Thoughts on how to test this? I can't get the test to replicate the bug. I've tried adding pointer events on a sibling element, container, document, and on window

Reply:
I have tried few cases myself but nothing worked. Maybe it is impossible to reproduce in jsdom.

Reply:
Maybe we should complete this PR without a test? 😄

Reply:
We might skip tests for this one then. I am just thinking whether it is a good solution because it is always listening for the document events, I kind of like the way you did in Info Panel:
![image](https://user-images.githubusercontent.com/36186912/151796753-9ffbfd79-14fe-44b7-ab05-fe2e7db871a8.png)

Reply:
I tried doing that and it got too complicated and was breaking things, so I reverted it